{{cookiecutter.project_name}}/.github/workflows/ci-pipeline.yaml

```diff
@@ -12,43 +13,46 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v3
-    - name: Install dependencies
-      run: |
-        python -m pip install -r .github/workflows/requirements.txt
-    - name: Lint Code Base
-      uses: github/super-linter@v4
-      env:
-        VALIDATE_ALL_CODEBASE: false
-        DEFAULT_BRANCH: "main"
-        VALIDATE_MARKDOWN: true
-        VALIDATE_PYTHON_PYLINT: true
-        VALIDATE_PYTHON_FLAKE8: true
-        VALIDATE_TERRAFORM_TFLINT: true
-        VALIDATE_YAML: true
-
-    - name: Detect Secrets Dependencies
-      run: |
-          apt-get update && apt-get install -y jq
-          pip install yq
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install dependencies
+        run: |
+          python -m pip install -r .github/workflows/requirements.txt
+
+      - name: Lint Code Base
+        uses: github/super-linter@v4
+        env:
+          VALIDATE_ALL_CODEBASE: false
+          DEFAULT_BRANCH: "main"
+          VALIDATE_MARKDOWN: true
```

Comment:
do we need the markdown and yaml super linter if we added the ones in the linter folder?

Reply:
The files in the linter folder are only the exception files for superlinter, so that is still needed.